certified-connectors/Document Merge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,191 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "DocumentMergeWebAPI",
```

Comment:
@cirrussoft do you want to change the title? It was Document Merge in the previous commit, and this variant contains API in the title, it's not allowed. More information about connector title can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title).

Reply:
Fixed the issue